Index: server/WS-events/test-client.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/server/WS-events/test-client.py b/server/WS-events/test-client.py
new file mode 100644
--- /dev/null	(date 1692473545784)
+++ b/server/WS-events/test-client.py	(date 1692473545784)
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+import json
+
+from websockets.sync.client import connect
+
+
+def hello():
+    with connect("ws://localhost:8080") as websocket:
+        websocket.send(json.dumps({
+            'test': True,
+            'debug': 0
+        }))
+
+
+hello()
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"ProjectRootManager\" version=\"2\" project-jdk-name=\"Python 3.11 (GigaChat)\" project-jdk-type=\"Python SDK\" />\r\n  <component name=\"PyCharmProfessionalAdvertiser\">\r\n    <option name=\"shown\" value=\"true\" />\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	(revision 92762252399531eefea31089530ccf3f6a14983b)
+++ b/.idea/misc.xml	(date 1692473545784)
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.11 (GigaChat)" project-jdk-type="Python SDK" />
+  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.10 (GigaChat) (2)" project-jdk-type="Python SDK" />
   <component name="PyCharmProfessionalAdvertiser">
     <option name="shown" value="true" />
   </component>
